.github/workflows/nixf-tidy-action.yaml

```diff
@@ -0,0 +1,25 @@
+name: nixf-tidy code linter
+
+on:
+  [ pull_request ]
+
+jobs:
+  nixf-tidy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+
+    if: ${{ !contains(github.event.head_commit.message, '[skip ci]') }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: cachix/install-nix-action@v22
+      with:
+        github_access_token: ${{ secrets.GITHUB_TOKEN }}
+        extra_nix_config: |
+          trusted-public-keys = nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+          substituters = https://nix-community.cachix.org https://cache.nixos.org/
+    - run: nix profile install github:nix-community/nixd#nixd
```

Comment:
IMO this should be the responsibility of the `nixf-tidy-action` action.

Is there some technical reason why _we_ need to install the action's dependency in _our_ workflow?

Reply:
> IMO this should be the responsibility of the `nixf-tidy-action` action.
> 
> Is there some technical reason why _we_ need to install the action's dependency in _our_ workflow?

Thanks, I just wrote `nixf-tidy-action` action in purely typescript. Thus I think it is more elegant to handle dependencies in nix?

(Otherwise these installation should be performed in "js" exec, looks bad!)